### PR TITLE
Simplify build of the Flexray Agent

### PR DESCRIPTION
**Pull request by enmasse**
Build the Flexray agent with Travis

### CODE REVIEW

**Comment by AleksandarFilipov** [comment]
* As you already started i would suggest that you replace "wice" everywhere, including in filenames.
* instructions on how to build on arm32 (raspberry pi) would be a good step before the project the build as automated.

**Comment by enmasse** [comment]
> As you already started i would suggest that you replace "wice" everywhere, including in filenames.
> instructions on how to build on arm32 (raspberry pi) would be a good step before the project the build as automated.

Done

File: tools/flexray2ip/Makefile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+CC=gcc
+CFLAGS=-I. -std=c99 -D_DEFAULT_SOURCE
+DEPS = flexray.h
+.DEFAULT_GOAL := all
+
+%.o: %.c $(DEPS)
+	$(CC) -c -o $@ $< $(CFLAGS)
+
+wice-flexray2ip: flexray2ip.o
```
Comment (AleksandarFilipov) [inline tools/flexray2ip/Makefile]: would the following change make sense?
```suggestion
flexray2ip: flexray2ip.o
```

File: tools/flexray2ip/README.txt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Building
+--------
+
+You can typically build the Flexray agent in an environment with Gnu Make and GCC.
+To be able to run the binary on the Host Mobility MX-4 you need to compile it for
+the armv7 architecture. The easiest way to do that if you're inexperienced is to build
+it on a Raspberry Pi with Raspbian.
+
+In that case it's just a matter of running "make" in this directory.
```
Comment (AleksandarFilipov) [inline tools/flexray2ip/README.txt]: I'll try and see if this works ;) Let me try before we merge.

Comment (AleksandarFilipov) [reply tools/flexray2ip/README.txt]: Yep building works! (i'll try on a hardware later on)

**Pull request merged**